checks/libs/wmi/sampler.py

```diff
@@ -91,8 +92,10 @@ class WMISampler(object):
 
     def __init__(self, logger, class_name, property_names, filters="", host="localhost",
                  namespace="root\\cimv2", provider=None,
-                 username="", password="", and_props=[], timeout_duration=10):
+                 username="", password="", and_props=[],
+                 mute=True, timeout_duration=10):
```

Comment:
I get why you have implemented the `mute` feature. But do we really need it? None of the checks initiate the sampler with a different value here, no?